packages/math-input/src/full-math-input.stories.tsx

```diff
@@ -6,16 +7,38 @@ import {KeypadInput, KeypadType, MobileKeypad} from "./index";
 
 export default {
     title: "Full Mobile MathInput",
+    parameters: {
+        backgrounds: {
+            default: "light background",
+            values: [
+                // We want a slightly darker default bg so that we can
+                // see the top of the keypad when it is open
+                {name: "light background", value: "lightgrey", default: true},
+            ],
+        },
+        viewport: {defaultViewport: "iphone6", viewports: INITIAL_VIEWPORTS},
+    },
 };
 
 export const Basic = () => {
     const [value, setValue] = React.useState("");
     // Reference to the keypad
     const [keypadElement, setKeypadElement] = React.useState<KeypadAPI>();
     // Whether to use Expression or Fraction keypad
-    const [expression, setExpression] = React.useState<boolean>(true);
+    const [expression, setExpression] = React.useState<boolean>(false);
     // Whether to use v1 or v2 keypad
-    const [v2Keypad, setV2Keypad] = React.useState<boolean>(false);
+    const [v2Keypad, setV2Keypad] = React.useState<boolean>(true);
+    // Whether the keypad is open or not
+    const [keypadOpen, setKeypadOpen] = React.useState<boolean>(false);
+
+    const toggleKeypad = () => {
+        if (keypadOpen) {
+            keypadElement?.dismiss();
+        } else {
+            keypadElement?.activate();
+        }
+        setKeypadOpen(!keypadOpen);
+    };
```

Comment:
Was this just a convenience thing or was it not popping up for you when you focused on the input?

I'm a little worried that the state may get out of sync. It's a Story, so not a huge deal, but what if:

- Click the button to open: `keypadOpen === true`
- You click in the input
- You click outside the input, the `onBlur` closes the keypad
- Is `keypadOpen === true` still?

Reply:
This was a convenience thing for testing, as the keypad wasn't popping up for me otherwise :(